### PR TITLE
Add a pipeline to automate chart release

### PR DESCRIPTION
**Pull request by fonzdm**
* **Please check if the PR fulfills these requirements**
- [X] The branch naming convention follows our guidelines
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

This PR introduces a new workflow.

* **What is the current behavior?** (You can also link to an open issue here)

No automated mechanism is present, so releases and helm packages should be created manually.

* **What is the new behavior (if this is a feature change)?**

With this PR, a workflow to automatically create and publish releases is introduced.
The workflow will rely upon GitHub Pages and will push the generated helm charts into GHRC.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

No. 

* **Other information**:

#3 #8

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+      - ci/publish-pipeline
```
Comment (imgios) [inline .github/workflows/release.yaml:7]: Can we remove this branch and leave just the `main` one?

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+          helm repo add truecharts https://charts.truecharts.org
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.5.0
+        with:
+          charts_dir: ./
+          config: ./.github/ci/cr.yaml
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          CR_SKIP_EXISTING: "false"
```
Comment (imgios) [inline .github/workflows/release.yaml:44]: What if we set this to true?

**Review by imgios — COMMENTED** [review]
Hi @fonzdm, this looks good but I think there are a couple of things to review. Let me know your POV.

**Comment by imgios** [comment]
I also have another question: should we also update the deployment section in the README.md as part of this PR? I think we can add the deployment method by adding the GitHub Pages instance as Helm Repository.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (fonzdm) [reply .github/workflows/release.yaml]: done

File: .github/workflows/release.yaml (hunk 2 above)
Comment (fonzdm) [reply .github/workflows/release.yaml:43]: You are right, we left it false just to make a test. Gonna replace it

**Comment by fonzdm** [comment]
> I also have another question: should we also update the deployment section in the README.md as part of this PR? I think we can add the deployment method by adding the GitHub Pages instance as Helm Repository.

You are right! I will make this change asap

**Pull request merged**